Sources/GRPCHealthService/HealthService+Service.swift

```diff
@@ -92,6 +110,10 @@ extension HealthService.Service {
       }
     }
 
+    fileprivate func listStatuses() -> [String: Grpc_Health_V1_HealthCheckResponse.ServingStatus] {
+      self.lockedStorage.withLock { $0.mapValues(\.currentStatus) }
```

Comment:
Avoid using key-paths: they're a _lot_ slower than just accessing the property

```suggestion
      self.lockedStorage.withLock { $0.mapValues { $0.currentStatus } }
```